src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -5,11 +5,19 @@ import { MessagePackHubProtocol } from '@aspnet/signalr-protocol-msgpack';
 import { OutOfProcessRenderBatch } from './Rendering/RenderBatch/OutOfProcessRenderBatch';
 import { internalFunctions as uriHelperFunctions } from './Services/UriHelper';
 import { renderBatch } from './Rendering/Renderer';
-import { fetchBootConfigAsync, loadEmbeddedResourcesAsync } from './BootCommon';
+import { fetchBootConfigAsync, loadEmbeddedResourcesAsync, autoStartIfApplicable } from './BootCommon';
 import { CircuitHandler } from './Platform/Circuits/CircuitHandler';
 import { AutoReconnectCircuitHandler } from './Platform/Circuits/AutoReconnectCircuitHandler';
 
-async function boot() {
+let started = false;
+
+async function boot(options?: any) {
+
+  if (started) {
+    throw new Error('Blazor has already started.');
+  }
+  started = true;
+
   const circuitHandlers: CircuitHandler[] = [ new AutoReconnectCircuitHandler() ];
   window['Blazor'].circuitHandlers = circuitHandlers;
```

Comment:
I took no steps to address the fact that this script relies on mutating global state when `boot()` runs. It seems a little odd to me. LMK if you think this needs to be addressed as part of this change.

Reply:
We should file a bug for it and not do anything right now. In the end I will expect for boot to take in some sort of options. Or to have a builder pattern to configure things a la signalr

Reply:
This is that PR though.